bcipy/helpers/copy_phrase_wrapper.py

```diff
@@ -134,26 +171,53 @@ def evaluate_inquiry(self, raw_data, triggers, target_info, window_length):
                                     channel_map=self.channel_map,
                                     trial_length=window_length)
 
-        lik_r = self.signal_model.predict(x, letters, self.alp)
-        prob = self.conjugator.update_and_fuse({'ERP': lik_r})
-        decision, sti = self.decision_maker.decide(prob)
+        return self.signal_model.predict(x, letters, self.alp)
 
-        return decision, sti
+    def add_evidence(self, name: str, evidence: List[float]) -> np.array:
```

Comment:
Could this use the new `EvidenceName`?

Reply:
It's not a true Enum, but maybe I should refactor so it is to be more explicit about types.